core/src/main/scala/cats/collections/TreeList.scala

```diff
@@ -658,12 +659,13 @@ object TreeList extends TreeListInstances0 {
           else {
             val h1 = h.updated(idx, a)
             // now rebuild the front of the list
-            Trees(front.reverse_:::(h1 :: tail))
+            Trees(front reverse_::: (h1 :: tail))
```

Comment:
To someone who interested in that fix - we've added `Scalafmt` with the `AvoidInfix` rewriting rule. So `Scalafmt` formatted `front reverse_::: (h1 :: tail)` as `front.reverse_:::(h1 :: tail)`. But that leads to behaviour changes for `reverse_:::` (see https://www.scala-lang.org/api/2.13.6/scala/collection/immutable/List.html#reverse_:::[B%3E:A](prefix:List[B]):List[B]).